docs/source/examples/tabular/checks/integrity/source/plot_dominant_frequency_change.py

```diff
@@ -2,43 +2,58 @@
 """
 Dominant Frequency Change
 *************************
-"""
+This example provides an overview for using and understanding the `Dominant Frequency Change` check.
 
-#%%
-# Imports
-# =======
+**Structure:**
 
-import pandas as pd
-from sklearn.datasets import load_iris
-from sklearn.model_selection import train_test_split
-from deepchecks.tabular.checks.integrity import DominantFrequencyChange
-from deepchecks.tabular import Dataset
+* `What is a Dominant Frequency Change? <#what-is-a-dominant-frequency-change>`__
+* `Generate Data <#generate-data>`__
+* `Run The Check <#run-the-check>`__
+* `Define a Condition <#define-a-condition>`__
 
-#%%
-# Generating Data
-# ===============
+What is a Dominant Frequency Change?
+====================================
+Dominant Frequency Change is a data integrity test which simply checks whether dominant values have increased
+significantly between the test data and a reference data.
```

Comment:
```suggestion
significantly between test data and train data.
```
I agree it doesn't have to be train, but we do call it train, so we should be consistent